### PR TITLE
master

### PR DESCRIPTION
**Pull request by DovAmir**
mode ui improvments:
show container status (running/faild)
show many containers in a node clearly
click container to see all container data (currently works only in master )

### CODE REVIEW

**Comment by DovAmir** [comment]
all .sh files should be removed

File: .idea/uiDesigner.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ManoMarks) [inline .idea/uiDesigner.xml]: @DovAmir last time we determined this file wasn't required. Can we leave it out?

File: src/utils/request.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ function filterStoppedTasks (objects) {
   let runningTasks = [];
   for(let i=0;i<objects.length;i++){
     let object = objects[i];
-    if(object.Status.State=="running" && object.DesiredState=="running") {
+    if( object.DesiredState=="running") { //object.Status.State=="running" &&
```
Comment (ManoMarks) [inline src/utils/request.js]: Did you mean to comment out object.Status.State @DovAmir?

File: .idea/uiDesigner.xml (hunk 1 above)
Comment (DovAmir) [inline .idea/uiDesigner.xml]: yes sorry about that, i thought it was in .gitignore
the .idea directory and the *.sh files should not be there

File: src/utils/request.js (hunk 2 above)
Comment (DovAmir) [inline src/utils/request.js]: Yes, If a containers DesiredState is running, but its actual  state is failed, I want it to appear in the UI with a red failed state,

**Pull request merged**